sound/soc/intel/boards/sof_sdw.c

```diff
@@ -895,6 +912,9 @@ static int create_sdw_dailink(struct snd_soc_card *card,
 	if (codec_info_list[codec_index].ignore_pch_dmic)
 		*ignore_pch_dmic = true;
 
+	if (is_amp(codec_info_list[codec_index].part_id) && link_id < SDW_AMP_DAI_ID)
```

Comment:
it would be simpler to add a field in codec_info_list, so that we have all the information in a single table.

we could even unify the rt715 handling with different flags for smart amp and smart mic.

Reply:
> it would be simpler to add a field in codec_info_list, so that we have all the information in a single table.
> 
> we could even unify the rt715 handling with different flags for smart amp and smart mic.

Done